frontend/src/index.js

```diff
@@ -17,3 +17,10 @@ ReactDOM.render(
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: http://bit.ly/CRA-PWA
 serviceWorker.unregister();
+
```

Comment:
Can we remove this? (I know its from the original firebase branch but let;s not add it in)